docs/src/pages/[platform]/connected-components/storage/fileuploader/index.page.mdx

```diff
@@ -1,6 +1,6 @@
 ---
-title: Fileuploader
-description: The Fileuploader lets you upload files. 
+title: File Uploader
+description: The File Uploader lets you upload files to your S3 service! 
```

Comment:
feels a little too exited for a description 😂 

```suggestion
description: The File Uploader lets you upload files to your S3 service.
```

Reply:
👍